Xamarin.Forms.Platform.Android/Extensions/TextViewExtensions.cs

```diff
@@ -33,9 +33,9 @@ static void SetMaxLines(this TextView textView, Label label, int lines)
 			textView.SetMaxLines(lines);
 		}
 
-		public static void SetLineBreakMode(this TextView textView, Label label)
+		public static void SetLineBreakMode(this TextView textView, Label label, Button button = null)
```

Comment:
ABI breaking change

Reply:
@samhouts even with `button` being null by default is an API breaking change?

Reply:
Well, it's technically not a breaking change yet, but an optional parameter here feels a bit strange. I think I'd rather see two methods here with some code duplication or some refactoring to call a third common method.